backend/portal/api/portal-api.yml

```diff
@@ -418,6 +418,54 @@ paths:
         "400":
           $ref: "#/components/responses/400"
 
+  /v1/user-collections/index:
```

Comment:
is this route essentially returning the same stuff as [this route](https://github.com/chanzuckerberg/single-cell-data-portal/blob/6bf761664ad4e0db8157dd36392e8f64653d79b0/backend/portal/api/portal-api.yml#L16-L17) except for a narrower selection of Collections (i.e., only those for which the user is authorized)? If these routes serve the same general purpose, we should consider collapsing them so that you merely pass a query param to the existing route: `GET /v1/collections?owner=true` or similar.

Reply:
The old route that has been there forever is pretty poorly-defined in terms of response object... 😞